src/core/i18n/de/translation.json

```diff
@@ -1,13 +1,21 @@
 {
-  "intro_text": "Vielen Dank, dass Sie sich die Zeit nehmen, um diesen Fragebogen auszufüllen! Zuerst werden wir Sie bitten, alles zu beschreiben, was Ihr Kind gestern gemacht hat und zwar von dem Moment an, als ihr Kind gestern morgen aufgewacht ist bis zu dem Moment, als Ihr Kind gestern Nacht schlafen gegangen ist. Dann werden wir danach fragen, wie Ihr Kind letzte Nacht geschlafen hat. Bitte beantworten Sie die Fragen so gut Sie können. Am Ende der Umfrage werden Sie die Möglichkeit haben, Veränderungen vorzunehmen, wenn Sie möchten.",
-  "intro_text_ctx": {},
+  "intro_text": "$t(intro_text_ctx.intro)",
+  "intro_text_ctx": {
+    "intro_today": "Thank you for taking the time to complete this survey! First, we will ask how your child slept last night. Then, we will ask you to describe everything your child did today, starting when your child woke up this morning and ending when they went to bed tonight. Please answer the questions to the best of your ability. At the end of the survey, you will be able to make changes if you want.",
+    "intro_yesterday": "Vielen Dank, dass Sie sich die Zeit nehmen, um diesen Fragebogen auszufüllen! Zuerst werden wir Sie bitten, alles zu beschreiben, was Ihr Kind gestern gemacht hat und zwar von dem Moment an, als ihr Kind gestern morgen aufgewacht ist bis zu dem Moment, als Ihr Kind gestern Nacht schlafen gegangen ist. Dann werden wir danach fragen, wie Ihr Kind letzte Nacht geschlafen hat. Bitte beantworten Sie die Fragen so gut Sie können. Am Ende der Umfrage werden Sie die Möglichkeit haben, Veränderungen vorzunehmen, wenn Sie möchten."
+  },
   "choose_format": "Um zu beginnen, wählen Sie bitte aus, welches Uhrzeitenformat Sie bevorzugen.",
   "clock_formats": [
     "12-Stunden-Format",
     "24-Stunden-Format"
   ],
-  "day_of_week": "Wir möchten Sie bitten, über den Tag Ihres Kindes nachdenken und das Zeittagebuch für gestern auszufüllen. Welcher Wochentag war gestern?",
-  "day_of_week_ctx": {},
+  "today": "heute",
+  "yesterday": "gestern",
+  "day_of_week": "Wir möchten Sie bitten, über den Tag Ihres Kindes nachdenken und das Zeittagebuch für {{activityDay}} auszufüllen. Welcher Wochentag $t(day_of_week_ctx.what_dow)?",
```

Comment:
So sorry to find another tiny mistake here. This the corrected version: 
""Wir möchten Sie bitten, über den Tag Ihres Kindes nachzudenken und das Zeittagebuch für {{activityDay}} auszufüllen. Welcher Wochentag $t(day_of_week_ctx.what_dow)?"